src/operators.jl

```diff
@@ -858,3 +855,25 @@ function merge_vertices!(g::Graph{T}, vs::Vector{U} where U <: Integer) where T
 
     return new_vertex_ids
 end
+
+# special case for digraphs
+@traitfn function merge_vertices!(g::::IsDirected, v::Vector{U})  where U <: Integer
+    vs = Set(v)
+    v0 = min(vs...)
```

Comment:
Did you test the splat with `length(vs)` == 100M? Your testing should include very large graphs.

Reply:
Also, getting the `minimum()` of a vector is an order of magnitude faster than getting the minimum of a set.